src/layout/BaseLayout/BaseLayout.tsx

```diff
@@ -36,7 +35,7 @@ const BaseLayout = ({ children }: Props) => {
   // Check if pathname includes a model name - and then always collapse sidebar
   const { modelName } = useParams<EntityDetailsRoute>();
 
-  const collapseSidebar = useSelector<TSFixMe>(isSideNavCollapsed) || false;
+  const collapseSidebar = useSelector(isSideNavCollapsed) || false;
```

Comment:
I've migrated the utils and selectors to TypeScript so a bunch of these TSFixMes aren't required any more.